sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/constraints/BaseConstraint.java

```diff
@@ -73,6 +73,18 @@ public String toDDL() {
         rely ? "RELY" : "NORELY");
   }
 
+  public String toDescription() {
+    StringJoiner joiner = new StringJoiner(" ");
+    joiner.add(definition());
+    if (enforced) {
+      joiner.add("ENFORCED");
+    }
```

Comment:
Currently the ENFORCED behavior can't be set. The check constraint is always enforced while the other constraints are not enforced. We can either:
* Always show `ENFORCED`/`NOT ENFORCED`
* Skip this one in the toDescription method